avro_schema/frontend.lua

```diff
@@ -1086,29 +1104,36 @@ export_helper = function(node, already_built)
                 }
                 res.fields[i] = xfield
             end
+            pack_nullable_to_type(res)
             return res
         elseif xtype == "enum" then
             if already_built[node.name] then
-                return node.name
+                return get_packed_nullable_type(node)
```

Comment:
So, exporting schema where 'enum' is top-level element can contain 'nullable' field? Don't sure it is correct.